config/management/src/smoke_test.rs

```diff
@@ -120,6 +120,23 @@ fn smoke_test() {
     for (i, mut config) in configs.iter_mut().enumerate() {
         let ns = i.to_string();
         let waypoint = helper.create_waypoint(&ns).unwrap();
```

Comment:
you need to change this, we should create the waypoint outside this loop -- create it in the shared secure storage -- create should not be modifying any validators storage